### PR TITLE
[XChange] Add basic support for crypto currency ticker

### PR DESCRIPTION
**Pull request by tdiesler**
Resolves CAMEL-12066

### CODE REVIEW

File: components/camel-xchange/src/main/docs/weather-component.adoc [hunk 1]
```diff
@@ -0,0 +1,176 @@
+== Weather Component
```
Comment (oscerd) [inline components/camel-xchange/src/main/docs/weather-component.adoc]: Looks like the docs are from the weather component

Comment (tdiesler) [reply components/camel-xchange/src/main/docs/weather-component.adoc]: I updated the docs with an initial draft. Some of the sections seem to be generated & I would need a little help on how to do this. Also, I'd like to use the name part of the URI, which for some reason in null in the configuration. Perhaps you could have a look at what I might have done wrong here.

Comment (tdiesler) [reply components/camel-xchange/src/main/docs/weather-component.adoc]: Is there a way to mark the component as preview? The API is still expected to change a lot while I move along.

Comment (oscerd) [reply components/camel-xchange/src/main/docs/weather-component.adoc]: Actually you need to add a comment manually for the preview status. At first glance to generate the docs automatically with component and endpoint options, you need to name the .adoc with the component name, so xchange-component.adoc.

**Comment by davsclaus** [comment]
Yeah the documentation needs to be updated. The license of that 3rd party JAR is MIT so its acceptable for ASF.

**Comment by tdiesler** [comment]
The requested docs changes are done, I believe.

**Comment by oscerd** [comment]
Thanks, I'll merge soon :-)

**Comment by oscerd** [comment]
The PR has been merged. Thanks :-)

**Pull request closed without merging**